arduino/src/main.cpp

```diff
@@ -74,7 +83,7 @@ void loop() {
                 digitalWrite(LED, LOW);
                 delay(500);
                 buttonState = digitalRead(BUTTON);
-                httpResponseCode = http.sendRequest("POST", payload);
+                httpResponseCode = send_status_request("warsztat 027", BOARD_ID);

```

Comment:
Skoro masz nazwę miejsca zdefiniowaną już jako zmienną globalną to nie musisz znowu ręcznie podawać tego samego jako argumentu.

Reply:
Analogiczny problem jest z niepotrzebnym przekazywaniem zmiennej BOARD_ID, która też jest globalna